projects/wave-core-new/src/lib/layers/legend/legend-raster/mapping-raster-legend.component.ts

```diff
@@ -13,12 +28,20 @@ import {ColorBreakpoint} from '../../../colors/color-breakpoint.model';
     styleUrls: ['mapping-raster-legend.component.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MappingRasterLegendComponent<S extends MappingRasterSymbology> extends RasterLegendComponent<S> implements OnChanges {
+export class MappingRasterLegendComponent extends RasterLegendComponent implements OnInit, OnChanges {
```

Comment:
I guess you should rename it to `RasterLegendComponent`

Reply:
A `RasterLegendComponent` already exists and is s super type of this component. Should I just remove that one?

Reply:
I guess we only need one raster legend component, so yeah :+1: .

Reply:
72f61062e83a2d0ec29735e73af94b52c2704b2e

Reply:
and 06f3d2b2849f39676631c70ceba921a15575db23